lib/nets/network.py

```diff
@@ -114,10 +114,10 @@ def _crop_pool_layer(self, bottom, rois, max_pool=True):
 
     # affine theta
     theta = Variable(rois.data.new(rois.size(0), 2, 3).zero_())
-    theta[:, 0, 0] = (x2 - x1) / (width - 1)
-    theta[:, 0 ,2] = (x1 + x2 - width + 1) / (width - 1)
-    theta[:, 1, 1] = (y2 - y1) / (height - 1)
-    theta[:, 1, 2] = (y1 + y2 - height + 1) / (height - 1)
+    theta[:, 0, 0] = ((x2 - x1) / (width - 1)).view(rois.size(0))
+    theta[:, 0 ,2] = ((x1 + x2 - width + 1) / (width - 1)).view(rois.size(0))
+    theta[:, 1, 1] = ((y2 - y1) / (height - 1)).view(rois.size(0))
+    theta[:, 1, 2] = ((y1 + y2 - height + 1) / (height - 1)).view(rois.size(0))
 
```

Comment:
What about changing to view(-1). Looks cleaner in that way.

Reply:
I agree. It looks better!